### DISCUSSION TITLE
[A-L] (2023/24) Foglio 4 - Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#159] [Esercizi - Bertini] [answered]
![Screenshot 2023-10-23 185112](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/b7f0cdcc-acd2-4abf-902b-0ee2d7e7b4ac)

**Comment by daromacarlo** [comment]
Applicazione della formula di Bayes.
![Immagine 2023-11-10 154259](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/2583e7c0-4889-4daa-88a4-347ae59e533e)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Confermata in Classe
Siano $\mathbb{P}(A),\mathbb{P}(B),\mathbb{P}(C)$ le probabilità di un prodotto di essere prodotto rispettivamente dalla macchinetta A, B, C e siano $\mathbb{P}(A_d),\mathbb{P}(B_d),\mathbb{P}(C_d)$ le probabilità delle rispettive macchinette di produrre un prodotto difettoso.
- **(1)** $\mathbb{P}(difettoso)=\mathbb{P}(A)\mathbb{P}(A_d|A)+\mathbb{P}(B)\mathbb{P}(B_d|B)+\mathbb{P}(C)\mathbb{P}(C_d|C)=\mathbb{P}(A)\frac{\mathbb{P}(A_d\cap A)}{\mathbb{P}(A)}+\mathbb{P}(B)\frac{\mathbb{P}(B_d\cap B)}{\mathbb{P}(B)}+\mathbb{P}(C)\frac{\mathbb{P}(C_d\cap C)}{\mathbb{P}(C)}=$
$=\mathbb{P}(A_d\cap A)+\mathbb{P}(B_d\cap B)+\mathbb{P}(C_d\cap C)=0.4\cdot 0.02+0.1\cdot0.03+0.5\cdot0.04=0.031$

- **(2)** Sia $X$ l'evento di essere prodotto da una delle macchinette $\mathbb{P}(d|X)= \frac{\mathbb{P}(d \cap X)}{\mathbb{P}(X)}=\frac{\mathbb{P}(d)}{\mathbb{P}(X)}$ e per Bayes:
$$\mathbb{P}(A|d)=\frac{\mathbb{P}(A)\mathbb{P}(d|A)}{\mathbb{P}(A)\mathbb{P}(d|A)+\mathbb{P}(B)\mathbb{P}(d|B)+\mathbb{P}(C)\mathbb{P}(d|C)}=\frac{0.02\cdot 0.4}{0.031}$$
$$\mathbb{P}(B|d)=\frac{\mathbb{P}(B)\mathbb{P}(d|B)}{\mathbb{P}(B)\mathbb{P}(d|B)+ \mathbb{P}(A)\mathbb{P}(d|A)+\mathbb{P}(C)\mathbb{P}(d|C)}= \frac{0.03\cdot 0.1}{0.031}$$
$$\mathbb{P}(C|d)=\frac{\mathbb{P}(C)\mathbb{P}(d|C)}{\mathbb{P}(C)\mathbb{P}(d|C)+\mathbb{P}(A)\mathbb{P}(d|A)+\mathbb{P}(B)\mathbb{P}(d|B)}= \frac{0.5\cdot 0.04}{0.031}$$

**Comment by niccolo-fato** [comment]
![IMAGE 2024-02-15 21:05:30](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/73995334/0fb59a14-a88f-4fe6-8888-f9dc6867ebe0)
